client/templates/availability/availability_slot_local.js

```diff
@@ -0,0 +1,11 @@
+import UHR from 'uhr';
+
+Template.availabilitySlotLocal.helpers({
+  localSlotTime: function(){
+    const slot = Template.instance().data;
+    let result = UHR(slot.d, slot.h, slot.m, -1)
+    result.hour = result.hour < 10  ? `0${result.hour}` : result.hour;
+    result.minute = result.minute < 10  ? `0${result.minute}` : result.minute;
+    return result;
```

Comment:
shouldn't this function return a time rather than a result?

Reply:
The result of the function is an UHR, which is a time object. 
Something like: { day:Number, hour:Number, minute:Number }

Here is the package: https://libraries.io/npm/uhr/0.0.2

Reply:
then the variable could be `timeObject`